webknossos/webknossos/dataset/layer.py

```diff
@@ -261,13 +265,17 @@ def get_best_mag(self) -> MagView:
     def add_mag(
         self,
         mag: Union[int, str, list, tuple, np.ndarray, Mag],
-        block_len: int = 32,
-        file_len: int = DEFAULT_WKW_FILE_LEN,
+        chunk_size: Optional[Union[Vec3IntLike, int]] = None,  # DEFAULT_CHUNK_SIZE,
```

Comment:
Not sure if these functions should accept `Optional[Union[Vec3IntLike, int]]` or `Optional[Vec3IntLike]`